src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+
+html {
```

Comment:
Redundant empty line at the begging of the file
```suggestion
html {
```